### ISSUE TITLE
How to export this project as an unsigned .ipa?

### ISSUE THREAD

**Issue by baba420** [jvanakker/tvOSBrowser#52]
Instead of sending directly to the Apple TV, can I also export it as an ipa file?

**Comment by YesThatAllen** [comment]
I don't know about unsigned.. but I was able to export a signed .ipa file by selecting `Archive` from the `Product` menu, then using the Distribute -> Release Testing option from there.

I did hit a problem building, however.. 

```
Print: Entry, "CFBundleVersion", Does Not Exist
<PATH_SNIPPED/-appletvos/Browser.build/Script-9693F8451BF5A99E00077BAB.sh: line 3: File Doesn't Exist, Will Create: /Users/Shared/Development/tvOS-Browser/_Project/Browser/Info.plist + 1: syntax error: invalid arithmetic operator (error token is "'t Exist, Will Create: /Users/Shared/Development/tvOS-Browser/_Project/Browser/Info.plist + 1")
Command PhaseScriptExecution failed with a nonzero exit code
```

Which is odd because the path the version increment script was trying to pass to `PlistBuddy` does exist.. I can only assume it's some sandboxing issue.

However, I didn't need the build incremented, so I deleted the three lines in the `Run Script` step from the `Build Phases` tab:


<img width="1122" height="459" alt="Image" src="https://github.com/user-attachments/assets/96fc1908-a00b-4f08-8b5d-3e389c4c629e" />



and could export an .ipa archive.

**Comment by YesThatAllen** [comment]
Thanks for the work on the browser app! I'll be near my Apple TV to try it out in a couple days.

I had the same issue even after checking out your branch.

The script in question is:

```bash
/usr/libexec/PlistBuddy -c "Print CFBundleVersion" "${PROJECT_DIR}/${INFOPLIST_FILE}")
buildNumber=$(($buildNumber + 1))
/usr/libexec/PlistBuddy -c "Set :CFBundleVersion $buildNumber" "${PROJECT_DIR}/${INFOPLIST_FILE}""
```

The variables resolve to where I'd checked out the repo:

`/Users/Shared/Development/tvOS-Browser/_Project/Browser/Info.plist`

I'd think the variable would need to point in to the build folder, more like `CODESIGNING_FOLDER_PATH`

(Just that one change doesn't work, because `$INFOPLIST_FILE` is `browser/Info.plist`, but would need to resolve to simply `Info.plist` to work)  


In any case.. I can get a working build by removing the "Run Script" build phase, but I wonder.. are we just doing something wrong?

**Comment by jvanakker** [comment]
Can you check again with the latest version?

**Comment by YesThatAllen** [comment]
@baba420 the rest of your question is largely going to rely on how you build, if you have a paid Apple Developer account, etc.
That said, as of b0e064f I am able to use the Product menu to create an Archive, and can distribute from there. 🚀

**Comment by jvanakker** [comment]
> Minor issue.. when I'm using Xcode Version 26.3 (17C529), and set my own Team ID, it wants to add a local reference to the `README.mdown`
> 

No, this is because I changed it so it's now referring to the correct place in the folder structure. The older version had a confusing duplicate README.mdown in the repo.

**Comment by YesThatAllen** [comment]
Very good, thank you! I can confirm that as of 34d3fbb, building/exporting .ipa works smoothly.

Additionally, nice work on the app overall!
